app/stac_api/admin.py

```diff
@@ -159,7 +159,7 @@ def clean_text_geometry(self):
 
 
 @admin.register(Item)
-class ItemAdmin(admin.GeoModelAdmin):
+class ItemAdmin(admin.ModelAdmin):
```

Comment:
I'm just wondering why this is necessary and make sure that it's an intended change and not an accident ;)

Reply:
same question: maybe a short comment or hint to changelog would be helpful for all changes of this kind

Reply:
Thanks for the input, yes it was intended, as `GeoModelAdmin` is deprecated in version 5 and is replaceable with `ModelAdmin`.
I added mode information to the PR description regarding deprecation changes.